### PR TITLE
[DO NOT MERGE] Changes to landing page header

### PR DESCRIPTION
**Pull request by andysellick**
**DO NOT MERGE until Monday 19th July**

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

## What

Change the icons on the coronavirus landing page. Includes a conditional that checks the yml for a variable and only changes the page when that variable is present.

## Why

The guidance is changing on Monday. We want to deploy these changes smoothly and with this technique we only have to deploy https://github.com/alphagov/govuk-coronavirus-content/pull/605 for them to appear.

## Visual changes

Before | After
------- | ------
Screenshot 2021-07-16 at 13 47 03 | Screenshot 2021-07-16 at 13 46 53

Trello card: https://trello.com/c/9NX8Pzh5/501-update-icons-from-coronavirus-landing-page-header-update-content

### CODE REVIEW

**Review by kevindew — COMMENTED** [review]
This looks good. Are you happy for me to deploy it to integration to try out?

Something came to mind, which is absolutely not a thing to worry about now (and seems like a massive pain), but when we have SVG's like this that embed text should they be using the <text> element of SVG?

File: app/assets/stylesheets/views/_covid.scss [hunk 1]
```diff
@@ -303,6 +303,16 @@ $c19-landing-page-header-background: govuk-colour("blue");
   }
 }
 
+.covid__logos-post-19-july {
+  width: 100%;
+  max-width: 500px;
+  margin-bottom: govuk-spacing(4);
```
Comment (leenagupte) [inline app/assets/stylesheets/views/_covid.scss:309]: I think this needs a smaller margin-top setting. The logos seem a little far from the page header compared to the design.

Comment (leenagupte) [reply app/assets/stylesheets/views/_covid.scss:309]: Design:
Screenshot 2021-07-16 at 14 07 31

Actual:
Screenshot 2021-07-16 at 14 08 02

**Comment by leenagupte** [comment]
Superseded by: https://github.com/alphagov/collections/pull/2482 as the icons are being removed rather than replaced.

**Pull request closed without merging**